vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/VSVideo.java

```diff
@@ -0,0 +1,123 @@
+package edu.asu.diging.vspace.core.model.impl;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import edu.asu.diging.vspace.core.model.IVSVideo;
+
+@Entity
+public class VSVideo extends VSpaceElement implements IVSVideo{
```

Comment:
this class looks like almost the same as VSImage. Let's extract the common parts into a shared superclass